tests/Microsoft.Identity.Web.Test/TokenAcquisitionAuthorityTests.cs

```diff
@@ -309,5 +311,88 @@ public void MergeExtraQueryParameters_MergedOptionsNull_Test()
             // Assert
             Assert.Null(mergedDict);
         }
+
+        [Theory]
+        [MemberData(nameof(ManagedIdClientIdCases))]
```

Comment:
>[MemberData(nameof(ManagedIdClientIdCases))] [](http://example.com/codeflow?start=8&length=44)

I think this is overcomplicating the test because it's not a complex object. You can just pass a string, like the test on line 146.